features/voorletters.feature

```diff
@@ -12,20 +12,32 @@ Als één of meerdere voornamen uit één letter bestaan, dan volgt er na de let
 Wanneer na een voorletter zonder punt (voornaam had één letter) nog een andere voorletter volgt, wordt daartussen een spatie gezet.
 Als de rubriek Voornamen is gevuld met de standaardwaarde '.' (punt), Dan wordt geen extra (scheidings)punt toegevoegd; de inhoud van de attribuut voorletters is na afleiding Dan '.'
```

Comment:
deze opmerking kan ook uit de feature. zoals hieronder bij het voorbeeld is gezegd, is "." geen officiële standaardwaarde voor voornamen.

Reply:
done